packages/eslint-plugin-udemy/rules/import-blacklist/index.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+const blacklist = [
+    {
+        // For correct configuration
+        pattern: '^currencyformatter(?:\\.js)?$',
+        message: 'Please import from utils/currency-formatter/, not from currencyformatter',
+        exceptions: ['utils/currency-formatter(?:\\.spec)?\\.js$'],
+    },
+    {
+        // For improved JS bundling
+        pattern: '^lodash(?:\\.js)?$',
+        message: 'Please import from lodash/lib/, not from lodash directly',
```

Comment:
@kevinzang maybe tweak the message here? We allow things like `import mapValues from 'lodash/mapValues'`.

Reply:
Thanks! I've updated the messages.